lib/galaxy/objectstore/s3_multipart_upload.py

```diff
@@ -14,17 +14,8 @@
 
 from multiprocessing.pool import IMapIterator
 
-try:
-    from galaxy import eggs
-    eggs.require('boto')
-except ImportError:
-    pass
-
-try:
-    import boto
-    from boto.s3.connection import S3Connection
-except ImportError:
-    boto = None
+import boto
+from boto.s3.connection import S3Connection
```

Comment:
`boto` is not in `lib/galaxy/dependencies/requirements.txt` , so maybe it would be better to leave the try-except here with a `assert boto is not None, BOTO_IMPORT_MESSAGE` below?